.github/workflows/ci.yml

```diff
@@ -88,6 +88,7 @@ jobs:
   e2e:
     name: e2e tests (consul-image=${{ matrix.consul-image }})
     strategy:
+      fail-fast: false
```

Comment:
Is there some set of tests where keeping this is useful alongside `-tags e2e ./internal/commands/server --failfast` below, or was this just for debugging?

Reply:
I would argue that this is useful in general because it keeps us from having to rerun the whole matrix when one test fails, but open to discussion about it.

Reply:
Specifically in the case where the test is flaky and all the other tests already passed.